### ISSUE TITLE
Installing @angular/fire in a new Angular 16 project

### ISSUE THREAD

**Issue by francesco-buscicchio** [angular/angularfire#3353]
### Version info
Angular: 16.0.0

### How to reproduce these conditions

I created a new project in Angular 16, it was required that I install @angular/fire however when I proceed with ng add it goes to error reporting me that "was found but does not support schematics"

![Cattura](https://github.com/angular/angularfire/assets/81358077/933484d7-f16a-4f47-9584-d1e7a2fd0829)

So as reported in issue no. 3348 (https://github.com/angular/angularfire/issues/3348) I tried inserting the "overrides" property inside package.json. Before installing but without success

![Cattura](https://github.com/angular/angularfire/assets/81358077/c4220204-ca3a-420a-93d8-db6a3052957d)

### Expected behavior
That the package be installed and the configuration initialized

### Actual behavior
Adding the package goes into error

**Comment by johnfewell** [comment]
@francesco-buscicchio AngularFire 7.5 isn't compatible with Angular 16. [See this issue.](https://github.com/angular/angularfire/issues/3348)

**Comment by mnc12004** [comment]
Probably not the right way to go about it, but this worked for me.

I updated @angular/fire/package.json
`  "peerDependencies": {
    "@angular/common": "^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0",
    "@angular/core": "^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0",
    "@angular/platform-browser": "^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0",
    "@angular/platform-browser-dynamic": "^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0",
    "rxjs": "~6.6.0 || ^7.0.0",
    "firebase-tools": "^9.9.0 || ^10.0.0 || ^11.0.0"
  },`
  and added the suggested overrides option after dependencies and the app worked fine.

**Comment by timonohe** [comment]
Is there any information about the version, when this issue will be fixed?

**Comment by francesco-buscicchio** [comment]
> Is there any information about the version, when this issue will be fixed?

I'm not aware of it, but I hope it's as soon as possible

**Comment by virendrapalsingh** [comment]
Facing the same issue on new project. I hope it will be resolved soon.

**Comment by hello99world99** [comment]
It is about the angular 16

**Comment by Mikael-Sundstrom** [comment]
The folloing code in package.json works for me when i install with command `npm install @angular/fire` instead of `ng add @angular/fire`.
```json
"dependencies": {
	...
},
"overrides": { 
	"@angular/fire": {
		"@angular/common": "^16.0.1",
		"@angular/core": "^16.0.1",
		"@angular/platform-browser": "^16.0.1",
		"@angular/platform-browser-dynamic": "^16.0.1"
	}
},
```

**Comment by virendrapalsingh** [comment]
@Mikael-Sundstrom It worked! Thank you very much.

**Comment by Jbz797** [comment]
It works but I still have this bug that blocks me on Angular 14: https://github.com/angular/angularfire/issues/3290
I don't want to activate `skipLibCheck` and I can't modify the files by hand either because I go through a CI.

**Comment by georgelviv** [comment]
Any updates? it's blocking me from migrating to Angular 16

**Comment by Harris6144** [comment]
Is this project dead now then? As no one is updating it to the latest version of Angular

**Comment by jamesdaniels** [comment]
Fixed in 7.6.0, which should be released momentarily
